tests/components/rfxtrx/conftest.py

```diff
@@ -46,27 +63,50 @@ async def setup_rfx_test_cfg(
     return mock_entry
 
 
+@pytest.fixture(autouse=True)
+async def transport_mock(hass):
+    """Fixture that make sure all transports are fake."""
+    transport = Mock(spec=RFXtrxTransport)
+    with patch("RFXtrx.PySerialTransport", new=transport), patch(
+        "RFXtrx.PyNetworkTransport", transport
+    ):
```

Comment:
Is it possible not to patch globally but only where it is used?

Reply:
We never want these to try external access, so i think it make sense to do global patching